pages/linux/umask.md

```diff
@@ -0,0 +1,15 @@
+# umask
+
+> Display file creation mask or set file creation mask to octal value.
```

Comment:
I agree. What is better "Set file permissions for newly created files." or something like "Set file mode creation mask." ?